pip-service/connection.go

```diff
@@ -0,0 +1,56 @@
+package pip_service
```

Comment:
The package as I commented above should be auto generated and contain only single gRPC wrapper file. It would be better to move the connection manager to PDP library.
Also the manager pretends to play two roles. It implements connection pool and it acts as a resolver for pip names. None of the roles have been completely implemented here. To make it production ready I'd move (at least for now) resolver role to DNS. So in pip selector URL `pip:<name>/<path>` the `<name>` would be just domain name of instance where PIP runs.
As connection pool the code should oversee connections and handle failed connections to prevent PDP from hanging.